_data/compose-cli/docker_compose_run.yaml

```diff
@@ -150,6 +179,18 @@ options:
   description: |
     Run command with the service's ports enabled and mapped to the host.
   deprecated: false
+  hidden: false
+  experimental: false
+  experimentalcli: false
+  kubernetes: false
+  swarm: false
+- option: tty
+  shorthand: t
+  value_type: bool
+  default_value: "true"
+  description: Allocate a pseudo-TTY.
+  deprecated: false
+  hidden: true
```

Comment:
Also see my other comment w.r.t. `-i`; looks like we're not consistent. I _think_ `-i` is also enabled/disabled through the `-T` / `--no-TTY` option? Or does that only toggle `-t`?